README.md

```
@@ -8,86 +8,86 @@
-  - Live preview of 3D models while working in Visual Studio Code
-  - Render 2D and 3D to images
-    - [x] `SVG`
-    - [x] `PNG`
-  - Export 3D models to CAD files
-    - [x] `STEP`
-    - [x] `STL`
-    - [x] `3MF`
-    - [x] `ThreeJS`
-    - [x] `OBJ`
+-   Multiple OSes supported
+    -   [x] Windows
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)